coriolis_openstack_utils/actions/tenant_actions.py

```diff
@@ -334,6 +335,18 @@ def execute_operations(self):
             network['id'] for network in
             networks.list_networks(self._source_openstack_client,
                                    src_tenant_id)]
+        if self.payload['replicate_flavors']:
+            src_flavors = [flavor.id for flavor in
+                           self._source_openstack_client.nova.flavors.list(
+                               is_public=None)]
+            for src_flavor_id in src_flavors:
+                dest_client = self._destination_openstack_client
+                flavor_migration_action = flavor_actions.FlavorCreationAction(
+                    {'src_flavor_id': src_flavor_id},
+                    source_openstack_client=self._source_openstack_client,
+                    destination_openstack_client=dest_client,
+                     )
```

Comment:
Please no trailing parens (and either way it's over-indented), please put it in the same line as the last kwarg.